Sources/Mantis/Protocols/TransformDelegate.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  TransformDelegate.swift
+//  Mantis
+//
+//  Created by Richard Shane on 21/3/2024.
+//
+
+import Foundation
+
+protocol TransformDelegate: AnyObject {
+    func undoManager() -> UndoManager
+    func isUndoEnabled() -> Bool
+    func isRedoEnabled() -> Bool
+    func undo()
+    func redo()
+    func updateCropState(_ cropState: CropState)
+    func enableUndo(_ enable: Bool)
+    func enableRedo(_ enable: Bool)
+    func enableReset(_ enable: Bool)
+}
```

Comment:
I suggest to rename function names as below to make the purpose clearer
```Swift
func updateEnableStateForUndo(_ enable: Bool)
func updateEnableStateForRedo(_ enable: Bool)
func updateEnableStateForReset(_ enable: Bool)
```

Reply:
changes made